Jenkinsfile_CNP

```diff
@@ -56,10 +56,6 @@ def publishRpaContract() {
 }
 
 withPipeline(type, product, component) {
-  pipelineConf = config
-  disableLegacyDeployment()
-  enableAksStagingDeployment()
-  enableSlackNotifications('#civil_notifications')
```

Comment:
I'm a bit unclear on what you're doing in general in this PR, but this stood out to me.

Is this removal intentional?

Reply:
(and I mean specifically the one on `enableSlackNotifications`

Reply:
yes it should be all as defualt so doesn't need calling out.